recipes-ni/docker-functional-tests/docker-functional-tests.bb

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "Functional test of docker container capabilities on NILRT"
+HOMEPAGE = "https://docker.com"
```

Comment:
For NI-authored recipes, we generally set the `HOMEPAGE` to `https://github.com/ni/nilrt`.